config/config.go

```diff
@@ -932,7 +932,7 @@ type ConsensusConfig struct {
 	// height (this gives us a chance to receive some more precommits, even
 	// though we already have +2/3).
 	// NOTE: when modifying, make sure to update time_iota_ms genesis parameter
```

Comment:
[question] I'm confused on the relationship between `TimeoutCommit`, `TargetRoundDuration` and `time_iota_ms`. Does `time_iota_ms` still need to be updated after this PR? If not,

```suggestion
```

Reply:
In theory, time iota should do what we want by setting a consensus param for the minimum time between commits, but in practice, setting the value to something larger than the actual block time messes with all of the timings and breaks header verification. Specifically when we check the times of the headers, the times often creep into the future. I think this comment is either outdated or is still relevant is the block times are shorter than the time_iota value

iirc, time iota is being removed in the next version of tendermint. Regardless though, I'm typically in favor of not changing comments like this in this repo unless truly necessary, and instead addressing them upstream if they need to be changed.

Reply:
> iirc, time iota is being removed in the next version of tendermint. Regardless though, I'm typically in favor of not changing comments like this in this repo unless truly necessary, and instead addressing them upstream if they need to be changed.

Agreed, I didn't realize this comment also exists in vanilla tendermint/tendermint [here](https://github.com/tendermint/tendermint/blob/35581cf54ec436b8c37fabb43fdaa3f48339a170/config/config.go#L937).